docs/mainnet.md

```diff
@@ -0,0 +1,174 @@
+
+To run a node as a relayer, pass the `--relayer` flag to `swapd`:
+```bash
+./bin/swapd --env stagenet --eth-endpoint MAINNET_ENDPOINT --relayer
+```
+
+Note: the current fee sent to relayers is 0.009 ETH per swap. Subtract the gas cost from this to determine how much profit will be made. The gas required to do a relayer-claim transaction is `102048` gas. Multiply this by the transaction gas price for the gas cost. The gas price is set via oracle unless you manually set it with the `personal_setGasPrice` RPC call.
+
+## swapcli commands
+
+`swapcli` is used to interact with `swapd`, ie. for finding peers and offers on the network and making/taking swaps.
+
+Some useful commands are:
+* `swapcli balances`: check your ETH and XMR addresses and balances.
+* `swapcli ongoing`: check the status of all ongoing swaps.
+* `swapcli past`: see all your past swaps.
+* `swapcli get-offers`: see all your currently advertised offers.
+
+You can see all available commands with `swapcli -h`.
+
+## Monero Taker 
+
+1. Check your Ethereum address and balance and ensure your address is funded:
+```bash
+./bin/swapcli balances
+```
+
+2. Search for existing XMR offers using `swapcli`:
+```bash
+./bin/swapcli discover --provides XMR --search-time 3 --swapd-port 5001
+# [[/ip4/127.0.0.1/udp/9934/quic-v1/p2p/12D3KooWC547RfLcveQi1vBxACjnT6Uv15V11ortDTuxRWuhubGv /ip4/127.0.0.1/udp/9934/quic-v1/p2p/12D3KooWC547RfLcveQi1vBxACjnT6Uv15V11ortDTuxRWuhubGv]]
+```
+
+3. Query a returned peer as to how much XMR they can provide and their preferred exchange rate (replace `"--multiaddr"` field with one of the addresses returned in the above step):
+```bash
+./bin/swapcli query --peer-id 12D3KooWC547RfLcveQi1vBxACjnT6Uv15V11ortDTuxRWuhubGv
+# Offer ID=cf4bf01a0775a0d13fa41b14516e4b89034300707a1754e0d99b65f6cb6fffb9 Provides=XMR MinimumAmount=0.1 MaximumAmount=1 ExchangeRate=0.05
+```
+
+> Note: the exchange rate is the ratio of XMR:ETH price. So for example, a ratio of 0.05 would mean 20 XMR to 1 ETH. 
+
+4. a. Then, finding an offer you like, take the offer by copying the peer's multiaddress and offer ID into the command below. As well, specify how much GoETH you would like to provide, taking into account the offer's exchange rate and min/max XMR amounts.
+```bash
+./bin/swapcli take --peer-id 12D3KooWC547RfLcveQi1vBxACjnT6Uv15V11ortDTuxRWuhubGv \
+  --offer-id cf4bf01a0775a0d13fa41b14516e4b89034300707a1754e0d99b65f6cb6fffb9 --provides-amount 0.05
+# Initiated swap with ID=0
+```
+
+5. b. Alternatively, you can take the offer without getting notified of swap status updates:
```

Comment:
We didn't mention that the previous entry gets status updates. You could mention it at the end of the previous entry and let the user know that CTL-C cancels the status updates, but does not cancel the swap.